app/src/main/java/com/sun_asterisk/moviedb_49/data/source/remote/fetchdata/ParseUrlToJson.kt

```diff
@@ -0,0 +1,55 @@
+package com.sun_asterisk.moviedb_49.data.source.remote.fetchdata
+
+import com.sun_asterisk.moviedb_49.data.model.Movie
+import com.sun_asterisk.moviedb_49.utils.entry.MovieEntry
+import org.json.JSONObject
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.net.URL
+
+private const val TIME_OUT = 15000
```

Comment:
```suggestion
private val TIME_OUT = 15000
```
chuyển vào trong class e